easybuild/easyconfigs/l/Lmod/Lmod-6.3.7.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = "Lmod"
+version = "6.3.7"
```

Comment:
plz, plz, plz. update this to 6.4.1 (argument: `module available` now works... yeah, right)